docs/guides/flux-inspired-practice.md

```diff
@@ -19,7 +19,7 @@ const useBoundStore = create((set) => ({
 }))
 ```
 
-See [Splitting the store into separate slices](./typescript.md#slices-pattern) for how to define a store with separate slices.
+See [Splitting the store into separate slices](./guides/typescript.md#slices-pattern) for how to define a store with separate slices.
```

Comment:
This makes the url `guides/guides/`. I think the proper fix is to remove `.md`. Hope it works in GitHub.
```suggestion
See [Splitting the store into separate slices](./typescript#slices-pattern) for how to define a store with separate slices.
```

Reply:
You're absolutely right. I'll get this fixed when I'm back at my desk

Reply:
If it doesn't work in GitHub, we should keep it as is, and fix it in the doc site generation tool.

Reply:
With an absolute path (in this case `/zustand/guides/typescript`) it appears to work correctly. I don't think any changes will be needed to the website repo

Reply:
That's not acceptable. Breaks the link in https://github.com/chrisk-7777/zustand/blob/issue-1216-docs/docs/guides/flux-inspired-practice.md, which we would want to keep working.

Reply:
Let's fix the website, it's how we did in jotai too.

Reply:
It's in this branch: https://github.com/pmndrs/website/tree/docs

Reply:
Right, I see what you mean by working in Github in the above comment. 

Ok, I'll take a look at fixing it at the website end

Reply:
Sorry, I wasn't very clear about "working in GitHub"...

Reply:
I've made a PR over on the site here: https://github.com/pmndrs/website/pull/251
I'm not 100% sold on the approach, so I'm open to suggestions. I believe this is what you meant by fixing it on the site side of things

Reply:
the approach in general? or the approach in the PR?
yes, this is what i meant. the logic in PR can be improved as Cody commented.

Reply:
With https://github.com/pmndrs/website/pull/251 now merged into the site repo, this PR has been updated to fix any outstanding broken relative links. These links now work on both GH and the docs site